### PR TITLE
Automate Grafana and dependencies installation 

### PR DESCRIPTION
**Pull request by Mihaylov93**
* Improve docker-compose file to avoid manual setup of plugins and dashboards.
   * One command and run out-of-the box experience.
 
* Updated docs to reflect this.
    * Preserved old documentation as a manual install reference.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -25,96 +25,46 @@ img
 
 ## Instructions
 
-**Note:** This guide assumes that you have a Grafana already [installed][grafana-install-docs]
-and running in a supported environment but for demonstration purposes the guide will provide
-instructions for the Docker-based deployment as it is more portable and cross-platform.
+We've supplied a Docker Compose file that simplifies the installation and configuration of Grafana,
```
Comment (layoaster) [inline docs/index.md]: no contractions in formal writings

```suggestion
We have supplied a Docker Compose file that simplifies the installation and configuration of Grafana,
```

File: docs/manual-installation.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+![Datasource setup](./images/datasource-setup.png)
+5. Click on the ***Save & test*** button (ignore the warning message) and click on the Back button.
+
+### Importing Edge Analytics dashboards
+
+You can import the Edge Analytics dashboards that we have made available:
+
+* [Basic dashboard](files/provisioning/dashboards/ea-basic-dashboard.json)
+* [Advanced dashboard](files/provisioning/dashboards/ea-advanced-dashboard.json)
```
Comment (layoaster) [inline docs/manual-installation.md]:
```suggestion
* [Basic dashboard](./files/provisioning/dashboards/ea-basic-dashboard.json)
* [Advanced dashboard](./files/provisioning/dashboards/ea-advanced-dashboard.json)
```

File: docs/manual-installation.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+         Authorization
+         ```
+
+      2. Value: (Available on [System73 Portal][s73-portal])
+
+         ```text
+         Bearer <edge_analytics_api_key>
+         ```
+
+![Datasource setup](./images/datasource-setup.png)
```
Comment (layoaster) [inline docs/manual-installation.md:75]: you can indent this to have the image within the numbered list margins

File: docs/index.md (hunk 1 above)
Comment (Mihaylov93) [reply docs/index.md]: https://github.com/System73/edge-analytics-grafana-setup/compare/c6478c05cb180a4559393ebae98efcf89f07d387..5d2a2fc506cc6b45041c2e4d653827e764276e8e

**Review by layoaster — APPROVED** [review]

**Pull request merged**